### PR TITLE
Test authoring tools

### PR DESCRIPTION
**Pull request by ppcano**
Communicate the Test Builder and Browser Recorder as tools available for the k6 OSS community.

Screen Shot 2020-11-19 at 5 01 54 PM

Screen Shot 2020-11-19 at 5 08 56 PM

Screen Shot 2020-11-19 at 5 01 37 PM

### CODE REVIEW

File: src/data/markdown/docs/01 guides/03 Test authoring/02 Recording a session/01 Browser recorder.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+![Step 3](./images/Recording-a-test-script/step-3.png)
+
+4 - **Save your test script**
+
+Save the recorded script in any of your projects.
+
+If any **third party requests** are made during the recording, those requests will be filtered out by default because: 
+
+- These third-party requests will skew the percentiles of your performance results.
+- You may have no ability to impact the performance of the third-party service
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/03 Test authoring/02 Recording a session/01 Browser recorder.md:66]:
```suggestion
- You may not have the ability to impact the performance of third-party services
```

File: src/data/markdown/docs/01 guides/03 Test authoring/02 Recording a session/02 Har converter.md [hunk 2]
```diff
@@ -1,31 +1,22 @@
...
 
-In terms of load testing, recording usually refers to the process of creating a load test from the recording of a user session. The process looks like:
 
-1. Record a user or API session.
-2. Convert the recorded session into a test.
-3. Run the test.
+The HAR converter is an alternative to the [Browser recorder](/test-authoring/recording-a-session/browser-recorder). It generates a k6 script based on the HTTP requests included on a [HAR file](<https://en.wikipedia.org/wiki/HAR_(file_format)>).
 
-While not exclusive, it is common to use the recording when testing complex scenarios of websites or mobile applications. If you have to create a performance test that simulates a scenario with dozens or hundreds of requests, the recording could help you to be more productive in creating your test.
+> HAR is a file format used by all the browsers and various tools to export the recorded HTTP requests.
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/03 Test authoring/02 Recording a session/02 Har converter.md:9]:
```suggestion
> HAR is a file format used by all major browsers and various other tools to export recorded HTTP requests.
```

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/01 Test authoring.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: 'Test authoring'
+excerpt: ''
+---
+
+In testing, authoring refers to the test creation process. 
+
+k6 is, at its core, a **Code-based testing tool**. Besides, the k6 ecosystem provides a set of **Codeless tools** to speed up the test creation and facilitate collaboration across different roles.
```
Comment (simskij) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/01 Test authoring.md:8]:
```suggestion
k6 is, at its core, a **Code-based testing tool**. However, the k6 ecosystem provides a set of **codeless tools** to speed up the test creation and facilitate collaboration across different roles.
```

File: .vale/Vocab/docs/accept.txt [hunk 4]
```diff
@@ -123,4 +123,5 @@ untyped
 vendored
 walkthrough
 webpages
-wpnonce
\ No newline at end of file
+wpnonce
+Codeless
```
Comment (simskij) [inline .vale/Vocab/docs/accept.txt:127]:
```suggestion
Codeless
```
```suggestion
(?i)codeless
```

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/01 Test authoring.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Your own text editor or IDE
+- [Script editor in the k6 Cloud](/cloud/creating-and-running-a-test/script-editor)
+- [Test builder](/test-authoring/test-builder)
+- [Browser recorder](/test-authoring/recording-a-session/browser-recorder)
+- [Converters](/integrations#converters): JMeter, Postman and Swagger
+
+## Running a cloud test
+
+After creating your test, you might want to debug it locally or run it on the cloud. The cloud test will be launched from the geographic location/s - selected in your test - on the k6 Cloud infrastructure.
```
Comment (simskij) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/01 Test authoring.md:20]:
```suggestion
After creating your test, you might want to debug it locally or run it on the cloud. Cloud tests will be executed from the geographic location/s you've selected in your test, on k6 Cloud infrastructure. 
```

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/03 script editor.md [hunk 6]
```diff
@@ -1,23 +1,25 @@
 ---
-title: 'In-app script editor'
-excerpt: 'How to use the k6 in-app script editor'
+title: 'Script editor'
+excerpt: 'How to use the script editor in the k6 Cloud'
 ---
 
+The script editor is designed to allow you to quickly write and mock up test scripts right within the web app.
```
Comment (simskij) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/03 script editor.md:6]:
```suggestion
The script editor is designed to allow you to quickly write test scripts from within the web app. 
```

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/03 script editor.md [hunk 7]
```diff
@@ -1,23 +1,25 @@
 ---
-title: 'In-app script editor'
-excerpt: 'How to use the k6 in-app script editor'
+title: 'Script editor'
+excerpt: 'How to use the script editor in the k6 Cloud'
 ---
 
+The script editor is designed to allow you to quickly write and mock up test scripts right within the web app. 
+
+We believe most users will quickly graduate to using k6 as a CLI and their IDE of choice locally. However, organizational security rules vary from organization to organization and some may not be able to install a program without review from their security team. The script editor allows you to utilize just about all features of k6 right within the web app.
```
Comment (simskij) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/03 script editor.md:8]:
```suggestion
We believe most users will quickly graduate to using k6 as a CLI locally together with their IDE of choice. However, organizational security rules vary from organization to organization and some may not be able to install applications without a review from their security team. The script editor allows you to utilize just about all features of k6 right within the web app.
```

**Review by simskij — CHANGES_REQUESTED** [review]
Great changes! Some nitpicks

File: src/data/markdown/docs/01 guides/03 Test authoring/01 test builder.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+The k6 Test Builder allows you to utilize a graphical interface to create a k6 test.
+
+Based on your inputs, the test builder will automatically generate the k6 script for you. Then, you could copy the script and [run the test from the CLI](/getting-started/running-k6).
+
+> **Note**: you need a [k6 Cloud](/cloud) account to use the test builder. However, it is **free to use**, and you do not need an active paid subscription to utilize this feature.
+
+Although we strongly believe in a scriptable/code-based tool to get the most of your performance testing. A GUI based tool like the test builder could benefit you:
+
+- Speeding up the test creation.
+- Instructing with the [k6 API](/javascript-api).
```
[the remark this answers is not in the document]
Comment (ppcano) [reply src/data/markdown/docs/01 guides/03 Test authoring/01 test builder.md]: https://github.com/loadimpact/k6-docs/pull/153/commits/786e8a91870c91073b40cc6500db279eddefd30b

**Pull request merged**